core/mixins/LoadSupport.js

```diff
@@ -50,6 +51,10 @@ export function LoadSupport(C) {
              * @param {LoadSpec} [loadSpec] - Metadata about the underlying request
              */
             async loadAsync(loadSpec = {}) {
+                warnIf(
+                    (!isNil(loadSpec) && !isPlainObject(loadSpec)),
```

Comment:
it seems to me that we should not be giving isNil a pass?  Won't that just fail badly on line 60?
Its not ok to pass null in here either, and undefined should yield the assignment of the default {}.

Reply:
👍 Updated, changelog entry added to highlight the stricter behavior.